assembl/static2/js/app/components/debate/survey/question.jsx

```diff
@@ -160,13 +147,11 @@ Question.propTypes = {
   mutate: PropTypes.func.isRequired
 };
 
-const QuestionWithMutation = graphql(createPostMutation)(Question);
-
 const mapStateToProps = (state) => {
   return {
     debate: state.debate,
     contentLocale: state.i18n.locale
   };
 };
 
-export default connect(mapStateToProps)(QuestionWithMutation);
\ No newline at end of file
+export default compose(graphql(createPostMutation), connect(mapStateToProps), withScreenDimensions)(Question);
```

Comment:
I think that you change the order of composition here. The `createPostMutation` needs `contentLocale` prop